plugins/importexport/native/filter/NativeXmlPublicationFilter.inc.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+/**
+ * @file plugins/importexport/native/filter/NativeXmlPublicationFilter.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class NativeXmlPublicationFilter
+ * @ingroup plugins_importexport_native
+ *
+ * @brief Class that converts a Native XML document to a set of articles.
+ */
+
+import('lib.pkp.plugins.importexport.native.filter.NativeXmlPKPPublicationFilter');
+
+class NativeXmlPublicationFilter extends NativeXmlPKPPublicationFilter {
+	/**
+	 * Constructor
+	 * @param $filterGroup FilterGroup
+	 */
+	function __construct($filterGroup) {
+		parent::__construct($filterGroup);
+	}
+
+
+	//
+	// Implement template methods from PersistableFilter
+	//
+	/**
+	 * @copydoc PersistableFilter::getClassName()
+	 */
+	function getClassName() {
+		return 'plugins.importexport.native.filter.NativeXmlPublicationFilter';
+	}
+
+	/**
+	 * Get the method name for inserting a published submission.
+	 * @return string
+	 */
+	function getPublishedSubmissionInsertMethod() {
```

Comment:
Is this function used? I don't see it anywhere (and I think it's the same method name in all apps anyway).

Reply:
Fixed.